src/drive/mobile/modules/mediaBackup/duck/bugFix.spec.js

```diff
@@ -0,0 +1,97 @@
+import { fixMagicFolderName } from './bugFix'
```

Comment:
By the way `bugFix` is maybe not a great name :D `fixBackupFolderNamesBug` or something more explicit might be better.